astropy/units/astrophys.py

```diff
@@ -98,26 +98,21 @@
          doc="Rayleigh: photon flux")
 
 
-###########################################################################
-# MISCELLANEOUS
-
-# Some of these are very FITS-specific and perhaps considered a mistake.
-# Maybe they should be moved into the FITS format class?
-# TODO: This is defined by the FITS standard as "relative to the sun".
-# Is that mass, volume, what?
-def_unit(['Sun'], namespace=_ns)
-
-
 ###########################################################################
 # EVENTS
 
-def_unit((['ct', 'count'], ['count']),
+def_unit((['ct', 'count', 'DN'], ['count']),
```

Comment:
:+1: to adding `DN`. One question, though: should it really be the same as `count`? Or might it be better as an alias of `adu` (see below) or just be its own unit?

One reason I ask is that to me prefixes (like milliDN`) do not really make sense, but for `count` we're stuck with having them (which I think is a bit of a mistake, but too late to fix!).

Reply:
Good question!  I don't have a strong opinion.  Generally DN is synonymous with counts in my experience.  I think probably `adu` should be the same.  It is a count.  Same unit, different name.  Any reason they are different units in astropy?  Am I opening a can of worms?  😅 

Agree prefixes don't make sense.

Reply:
Oh, and I see you can have millicounts but not picocounts?  And I see why (%).  🤯 

Would be nice to put that horse back into the stable.  😂 

Anyway, if DN was on its own as a unit, could it least have an equivalency to `u.ct` and `u.adu`?  That would seem useful.  I see that currently `u.ct != u.adu`.

Reply:
Yes, I was wondering about removing the prefixes and seeing if anyone notices...

Reply:
:+1: for removing stupid prefixes. Maybe after a qiuck search on github for u.milicounts ;-)